projects/plugins/crm/admin/support/main.page.php

```diff
@@ -207,7 +207,7 @@
 			<div class="data-shared">
 				<b>Site URL:</b> <?php echo esc_html( $site_url ); ?><br>
 				<b>License:</b> <?php echo esc_html( $license_key ); ?><br>
-				<b>Site data:</b><?php echo wp_json_encode( $site_data ); ?><br>
+				<b>Site data:</b><?php echo wp_json_encode( $site_data, JSON_UNESCAPED_SLASHES | JSON_HEX_TAG | JSON_HEX_AMP ); ?><br>
```

Comment:
```suggestion
				<b>Site data:</b><?php echo esc_html( wp_json_encode( $site_data, JSON_UNESCAPED_SLASHES | JSON_HEX_AMP ) ); ?><br>
```
or maybe drop the `JSON_HEX_AMP` and use `htmlspecialchars()` instead of `esc_html()`, or maybe even do like p1764180563508319/1763997995.289799-slack-C05Q5HSS013 if you really want to get fancy.